weave-js/src/common/util/markdown.ts

```diff
@@ -20,7 +20,12 @@ import {blankifyLinks, shiftHeadings} from './html';
 
 // exported only for tests
 export const DEFAULT_SANITIZATION_SCHEMA = _.merge(gh, {
-  attributes: {'*': ['className', 'style']},
+  attributes: {
+    '*': ['className', 'style'],
+  },
+  protocols: {
+    src: ['http', 'https', 'data'],
```

Comment:
From a pure code perspective, this seems fine. However this definitely needs a security review - I’m not sure how secure it is to allow markdown to use arbitrary data (this is unfortunately not restricted to images). Could be fine, but definitely in the realm of “get a security expert to tell you” :slightly_smiling_face:.